example/48_pool3d_fwd/pool3d_fwd_fp16.cpp

```diff
@@ -15,43 +15,71 @@ using ComputeDataType = float;
 
 using IndexDataType = int32_t;
 
+#if 1
 using InLayout  = ck::tensor_layout::convolution::NDHWC;
 using OutLayout = ck::tensor_layout::convolution::NDHWC;
 
+static constexpr bool IsFastestDimReduced = false;
+#else
+using InLayout  = ck::tensor_layout::convolution::NCDHW;
+using OutLayout = ck::tensor_layout::convolution::NCDHW;
+
+static constexpr bool IsFastestDimReduced = true;
```

Comment:
Are you sure you can use `IsFastestDimReduced = true` when the layout is NCDHW ?